WordPress/src/test/java/org/wordpress/android/RobolectricSetupTest.java

```diff
@@ -14,7 +14,7 @@
 import org.robolectric.annotation.Config;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(application = TestApplication.class, sdk = VERSION_CODES.LOLLIPOP)
+@Config(application = TestApplication.class, sdk = VERSION_CODES.N)
```

Comment:
I'm guessing you bump to `N` here (24) as it is the new minimum SDK version?

Reply:
That's correct.